### ISSUE TITLE
Success icon not aligned in notification

### ISSUE THREAD

**Issue by alanzanattadev** [tuomashatakka/reduced-dark-ui#6]
Hi,

the success icon is aligned to the top instead of the center in the green part of the notification :)

Thank you

**Comment by tuomashatakka** [comment]
Could you send a screenshot? Shouldn't the icons be aligned to the top?

**Comment by alanzanattadev** [comment]
It does something like : 
------------------------------------------------------------------
| x |
|    | The notification message
|    |
|    |
------------------------------------------------------------------

instead of 

------------------------------------------------------------------
|      |
|      | The notification message
|  x  |   <= centered
|      |
------------------------------------------------------------------

Notice my ASCII arts skills very developed :D More seriously, I can't screenshot it I'm on linux and the time it takes to screenshot makes the notification disappear

**Comment by tuomashatakka** [comment]
Cool! (I'm referring to the ascii art 🙂) 

Hmm,  I think they are supposed to be on the top-left corner, at least I've had them there for as long as I remember. Tried to solve this but since it's done with a single pseudo element, it got a bit hacky at some point, so I gave up. I'd like to do something radical for the notifications anyway, for me they have always bugged me most atom-wise. Any ideas?
